assessment.sql

```diff
@@ -5,24 +5,53 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
-
+-- CREATE TABLE licensees(
+  --in: create_table/licensees.sql
+id SERIAL PRIMARY KEY,
+surname CHARACTER VARYING,
+given_name CHARACTER VARYING,
+gender CHARACTER (1),
+height INTEGER,
+weight INTEGER,
+dob DATE,
+eye_color CHARACTER VARYING,
+hair_color CHARACTER VARYING
+);
+--in psql --in psql: \i scripts/create_table/licensees.sql
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
-
--- your code here
+--in insert_data/licensees.sql--
+INSERT INTO licensees(surname,given_name, gender, height, weight, dob, eye_color, hair_color)
+VALUES (`Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black`);
+--in psql \i scripts/insert_into/licensees.sql
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
-
+-- bulk load licensee data
+\copy licensees(surname,given_name, gender, height, weight, dob, eye_color, hair_color)FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv)
+--psql \i scripts/bulk_load/licensees.sql
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
-
--- your code here
+--in psql:
+SELECT * FROM licensees WHERE eye_color IS 'Hazel', 'Brown','Black';
```

Comment:
`IS` is not used for comparing to values other than `NULL`. Commas also do not work for the purpose you're trying to use them for here. Finally, the question asks you to match `eye_color` to a certain value and also `hair_color` to certain values. Write the comparisons and chain them together with the correct logical operators to solve this question.

1/4